Formula/gmt.rb

```diff
@@ -1,11 +1,11 @@
 class Gmt < Formula
   desc "Tools for processing and displaying xy and xyz datasets"
```

Comment:
```suggestion
  desc "Tools for manipulating geographic and Cartesian data and creating plots, maps, and 3D views"
```
The old description isn't accurate.